src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -141,33 +140,25 @@ public void testNewCtorWithTwoParameters() {
     @Test
     public void testNullInfoStreamOptions() {
         final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        final IllegalArgumentException ex = assertThrows(IllegalArgumentException.class,
-                () -> new DefaultLogger(outputStream, (OutputStreamOptions) null),
-                "IllegalArgumentException expected");
-        assertWithMessage("Invalid error message")
-                .that(ex)
-                .hasMessageThat()
-                        .isEqualTo("Parameter infoStreamOptions can not be null");
+        final DefaultLogger logger =
+                new DefaultLogger(outputStream, (OutputStreamOptions) null);
+
+        final boolean closeError = TestUtil.getInternalState(logger, "closeError");
```

Comment:
Test should be same as testNewCtorWithTwoParameters() the only difference is null in ctor.

Reply:
I have done that except i made a slight modification by replacing IllegalStateException with IllegalArgumentException to satisfu the original condition of the unit test.